datadog_checks_dev/datadog_checks/dev/plugin/tox.py

```diff
@@ -11,6 +11,7 @@
 STYLE_CHECK_ENV_NAME = 'style'
 STYLE_FORMATTER_ENV_NAME = 'format_style'
 STYLE_FLAG = 'dd_check_style'
+STYLE_LOGGING_FORMAT = 'dd_check_style_logging_format'
```

Comment:
I think adding a new config for every new checker is not scalable. Let's change all checks then merge this into our standard style.

Reply:
> I think adding a new config for every new checker is not scalable.
For new checks, there is no need to add it since it should comply with those rules.

I added `dd_check_style_logging_format = false` for checks that does not comply yet.

Here is an example of "migrating" a check to comply with the new rules:
https://github.com/DataDog/integrations-core/pull/4716

> Let's change all checks then merge this into our standard style.
There is a risk of breaking things if we do it at once, since test coverage is not that high for some integrations.

Reply:
Then for now just add a new style command after isort and prefix with a `-` https://tox.readthedocs.io/en/latest/config.html#conf-commands

Reply:
Updated.